frontend/src/components/ExpenseList.js

```diff
@@ -10,7 +12,7 @@ class ExpenseList extends React.Component {
     return (
       <>
         <Form>
```

Comment:
This element had its own background color also: background: #E8A87C 11%;
Margin is also missing.

Reply:
Resolved in 82a9efe6dc53e25951eeecdd6b552f519d905b00